_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
+  variants: [ at, de, li, lu, ch ]
+
+-
+  code: el
+  name: Greek
+  family: grk
+  scripts: [ Grek ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ cy, gr ]
+
+-
+  code: en
+  name: English
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ au, bz, ca, 029, in, ie, jm, my, nz, ph, sg, za, tt, gb, us, zw ]
+
+-
+  code: eo
+  name: Esperanto
+  family: constructed language
+  scripts: [ Latn ]
+  typology:
+    - free word-order
+    - agglutinative
+  variants:
+
 -
   code: es
   name: Spanish
   family: roa
   scripts: [ Latn ]
-  typology: 
+  typology:
     - SVO
     - inflected
   variants: [ es, 419, ar, bo, cl, co, cr, do, ec, sv, gt, hn, mx, ni, pa, py, pe, pr, es, uy, ve, us ]
 
+-
+  code: et
+  name: Estonian
+  family: fiu
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative, fusional
+  variants: [ ee ]
+
+-
+  code: eu
+  name: Basque
+  family: isolating
+  scripts: [ Latn ]
+  typology:
+    - SOV
+    - agglutinative, fusional
+  variants: [ es ]
+
+-
+  code: fa
+  name: Persian
+  family: iir
+  scripts: [ Arab ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ af, ir ]
+
+-
+  code: fi
+  name: Finnish
+  family: fiu
+  scripts: [ Latn ]
+  typology:
+    - SVO, SOV
+    - synthetic, agglutinative
+  variants: [ fi ]
+-
+  code: fr
+  name: French
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ be, ca, fr, lu, mc, ch ]
+
+-
+  code: fy
+  name: Frisian
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ nl ]
+
+-
+  code: ga
+  name: Irish
+  family: cel
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - fusional
+  variants: [ ie ]
+
+-
+  code: gd
+  name: Scottish Gaelic
+  family: cel
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - inflected
+  variants: [ gb ]
+-
+  code: gl
+  name: Galician
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ es ]
+
+-
+  code: gu
+  name: Gujarati
+  family: inc
+  scripts: [ Gujr ]
+  typology:
+    - SOV
+    - inflected
+  variants: [ in ]
+
+-
+  code: ha
+  name: Hausa
+  family: afa
+  scripts: [ Latn, Arab ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ ng ]
+
+-
+  code: haw
+  name: Hawaiian
+  family: pqe
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - analytic
+  variants: [ us ]
+
+-
+  code: he
+  name: Hebrew
+  family: sem
+  scripts: [ Hebr ]
+  typology:
+    - VSO
+    - analytic, inflected
+  variants: [ il ]
+
+-
+  code: hi
+  name: Hindi
+  family: inc
+  scripts: [ Deva, Latn, Mahj ]
+  typology:
+    - SOV
+  variants: [ in ]
+
+-
+  code: hmn
+  name:  	Hmong
+  family: hmx
+  scripts: [ Latn, Hmng ]
+  typology:
+    - SVO
+    - analytic
+  variants:
+
+-
+  code: hr
+  name: Croatian
+  family: sla
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected, fusional
+  variants: [ hr, ba ]
+
+-
+  code: ht
+  name: Haitian
+  family: hat
```

Comment:
That's just the 3-letter code for the language.

The code for the family is `cpf` - French-based pidgins and creoles.